assets/blocks/course-outline/module-block/module.scss

```diff
@@ -39,11 +39,12 @@ $block: '.wp-block-sensei-lms-course-outline-module';
 	}
 
 	#{$block}__title {
+		color: inherit;
+		flex: 1;
 		font-size: 1.1em;
 		font-weight: inherit;
-		flex: 1;
 		margin: 0;
-		color: inherit;
+		padding-top: 0;
```

Comment:
This PR only adds this `padding`. The rest of the changes are just sorting the properties to follow the Sensei standard.